packages/lab/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -11,6 +11,10 @@ import linearProgressCss from "./LinearProgress.css";
 const withBaseName = makePrefixer("saltLinearProgress");
 
 export interface LinearProgressProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * A label for accessibility
+   */
+  "aria-label"?: string;
   /**
```

Comment:
Component changes should not be hidden in site PRs

Reply:
are you implying I should add a changeset or just not make this change in this PR?

Reply:
I would move this change from this PR and have it land in another. It's not clear why it was made

Reply:
I've removed the changes